resources/user-agents/crawlers/search-engines/petalbot.json

```diff
@@ -0,0 +1,30 @@
+{
+  "division": "PetalBot",
+  "sortIndex": 2415,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "PetalBot",
+      "browser": "petalbot",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "PetalBot"
+      },
+      "children": [
+        {
+          "match": "Mozilla\/5.0 (compatible; PetalBot;+https:\/\/aspiegel.com\/petalbot)",
+          "properties": {
+			"Comment": "PetalBot (PC)"
+          }
+        },
+        {
+          "match": "Mozilla\/5.0 (Linux; Android 7.0;) AppleWebKit\/537.36 (KHTML, like Gecko) Mobile Safari\/537.36 (compatible; PetalBot;+https:\/\/aspiegel.com\/petalbot)",
```

Comment:
Please make this rule more generic and not escape the Slashes.

Something like "Mozilla/5.0 (Linux; Android 7.*) AppleWebKit/* (KHTML, like Gecko) *Safari/* (compatible; PetalBot*"

We also have placeholder for platforms. Please look into the rules for the Chrome/Android, how to use.